Source/Schema.NET/ValuesJsonConverter.cs

```diff
@@ -169,6 +173,31 @@ public virtual void WriteObject(JsonWriter writer, object value, JsonSerializer
             serializer.Serialize(writer, value);
         }
 
+        private static Func<IEnumerable<object>, object> GetDynamicConstructor(Type objectType)

```

Comment:
Perhaps this can be pulled out into another class. Maybe something like `FastActivator.CreateInstance`.